mst/identify-index-issues.md

```diff
@@ -0,0 +1,96 @@
+---
+title: Identify and resolve issues with indexes in Managed Service for TimescaleDB
+excerpt: Identify and resolve issues with PostgreSQL indexes in Managed Service for TimescaleDB
+product: mst
+keywords: [index, issues, reindex]
+tags: [index, issues, REINDEX,]
```

Comment:
```suggestion
keywords: [index, REINDEX]
```

The related content function (which is what tags are for) also reads the keywords automatically, so there's no need to repeat tags that already in the keywords. Keywords are displayed on the user interface and tags are internal, so if you want any special capitalization, put the formatting on the keywords.